backend/users/models/user_edition.py

```diff
@@ -0,0 +1,23 @@
+from django.db import models
+from users.models.user import User
+from conspects.models.edition import Edition
+
+
+class PermissionType(models.TextChoices):
+    VIEW = 'view', 'View'
+    EDIT = 'edit', 'Edit'
+    OWNS = 'owns', 'Owns'
+    ADMIN = 'admin', 'Admin'
+
+
+class UserEdition(models.Model):
+    user = models.ForeignKey(User, on_delete=models.CASCADE, related_name='edition_permissions')
+    edition = models.ForeignKey(Edition, on_delete=models.CASCADE, related_name='user_permissions', null=True)
```

Comment:
I'm not sure if those should be ```on_delete=models.CASCADE```, wouldn't it delete edition and user if we take away permission from user with view/edit access?

Reply:
I tested removing permissions, but will test further and change if neccesary

Reply:
The on_delete=models.CASCADE argument in both ForeignKey fields (user and edition) does indeed specify that if a referenced User or Edition object is deleted, then the related UserEdition instances should also be deleted. However, it's important to clarify that this does not work the other way around: deleting a UserEdition instance does not trigger a deletion of the associated User or Edition. ~ Chat (i know that it's not the best source of info, but it makes sense)